x-pack/dockerlogbeat/docs/install.asciidoc

```diff
@@ -0,0 +1,109 @@
+[[log-driver-installation]]
+== Install and configure the {log-driver}
+
+++++
+<titleabbrev>Install and configure</titleabbrev>
+++++
+
+experimental[]
+
+[float]
+=== Before you begin
+
+Make sure your system meets the following prerequisites:
+
+* Docker: {docker-version}
+* {stack}: Version 7.6.0 and later
+
+//REVIEWERS: Are there any other prereqs?
+//REVIEWERS: Do we want to add a step for setting up a cloud trial? 
+
+[float]
+=== Step 1: Install the {log-driver} plugin
+
+// REVIEWERS: Will the plugin be available on docker hub when 7.6 ships, or will
+// users need to build from source? These instructions assume that the driver
+// will be available at the initial release. If it's not, we can add the steps
+// for building from source.
+
+// Related question: Do we want to document how to build/install from source
```

Comment:
@fearful-symmetry Sorry, I missed this one before. Do we want to also document how to build/install from source, or are the current instructions sufficient (at least for the initial release)?

Reply:
Actually, that might not be a bad idea. There's a single command, `mage BuildAndInstall`, that you can run from the `x-pack/dockerlogbeat` directory in order to build the plugin. It has the same build-time deps as most other beats: mage and docker.